torch/testing/_internal/common_fsdp.py

```diff
@@ -23,30 +33,97 @@
 
 
 class FSDPInitMode(Enum):
-    # Move model to CUDA before wrap
-    CUDA_BEFORE = 1
-    # Move model to CUDA after wrap
-    CUDA_AFTER = 2
-    # Don't move model to CUDA at all.
-    CUDA_NEVER = 3
-
-
-def _validate(model, process_group, assert_fn):
```

Comment:
Changed this to `_assert_module_states()` since `_validate()` was a bit non-descriptive.